src/README.md

```diff
@@ -18,18 +18,20 @@ As a Platform as a Service, or PaaS, Platform.sh automatically manages everythin
 
-You can use those concepts to replicate a traditional development/staging/production workflow, or to give every feature its own effective staging environment before merging to production (empowering you to use git-flow like methodologies even better). You could also have an intermediary integration branch for several other branches. Platform.sh respects the structure of branches. It’s entirely up to you.
+You can use these concepts to replicate a traditional development/staging/production workflow or even to give every feature its own effective staging environment before merging to production (empowering you to use git-flow like methodologies even better). You could also have an intermediary integration branch for several other branches.
+
+Platform.sh respects the structure of branches. It’s entirely up to you.
 
 ## Full stack management
 
 Managing your full stack internally gives Platform.sh some unique features:
 
-1. All of your services (MySQL, ElasticSearch, MongoDB, etc...) are managed inside the cluster and included in the price, with no external single-points-of-failure. So when you backup you get a fully consistent snapshot of your whole application.
-2. Multi-Services & Multi-App: you can deploy multiple applications (for example in micro-services style), using multiple data backends (MySQL, Postgres, Redis etc..) written in multiple frameworks (Drupal + NodeJS + Flask for example) in multiple languages in the same cluster.
-3. Full cluster cloning technology: the full production cluster can be cloned in under a minute, with all of its data to create on the fly ephemeral development environments that are a byte-level-copy of production, with all of its data.
-4. Fail-Proof Deployments: every time you test a new feature you also test the deployment process.
-5. Everything is build oriented with fully consistent and a repeatable build process, allowing you to easily keep your application updated and secure.
+1. **Unified Environment:** All of your services (MySQL, ElasticSearch, MongoDB, etc...) are managed inside the cluster and included in the price, with no external single-points-of-failure. When you backup an environment, you get a fully consistent snapshot of your whole application.
+2. **Multi-Services & Multi-App:** You can deploy multiple applications (for example, in a microservice-based architecture), using multiple data backends (MySQL, Postgres, Redis etc..) written in multiple frameworks (Drupal + NodeJS + Flask, for example) in multiple languages, all in the same cluster.
+3. **Full Cluster Cloning Technology:** The full production cluster can be cloned in under a minute&mdash;including all of its data&mdash;to create on-the-fly, ephemeral development environments that are a byte-level copy of production.
+4. **Fail-Proof Deployments:** Every time you test a new feature, you also test the deployment process.
```

Comment:
I think the comma on this line is extraneous.  (And I'm normally a comma dealer.)

Reply:
Which comma?

Reply:
"every time you test a new feature you also test the deployment process".  The comma after "feature" is IMO unnecessary.

Reply:
It wouldn't need a comma if it were "You also test the deployment process every time you test a new feature," but since "every time you test a new feature" precedes the subject of the sentence, it's functioning as an introductory phrase which should be set off by a comma.